src/fides/core/user.py

```diff
@@ -116,7 +121,11 @@ def create_command(
     )
     user_id = user_response.json()["id"]
     update_user_permissions(
-        user_id=user_id, scopes=SCOPES, auth_header=auth_header, server_url=server_url
+        user_id=user_id,
+        scopes=SCOPES,
+        auth_header=auth_header,
+        server_url=server_url,
+        roles=[ADMIN],
```

Comment:
Adjusting the CLI command to also assign all scopes and admin role here when creating a user with full permissions.